fineract-provider/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java

```diff
@@ -147,6 +151,34 @@ public CommandProcessingResult executeCommand(final CommandWrapper wrapper, fina
         return result;
     }
 
+    private String getIdempotencyKey(CommandWrapper wrapper) {
```

Comment:
Don't we want to extract this method to a separate class just for the sake of making this class easier to test?

Reply:
basically a few getters and setters but I will move it